src/elli_ws_request_adapter.erl

```diff
@@ -66,6 +66,13 @@
 
 -type req() :: #req_adapter{}.
 
+-ifdef(OTP_RELEASE). %% this implies 21 or higher
```

Comment:
Should we use something like this, as in ellli?

```erlang
{erl_opts, [{platform_define, "^2[1-9]", post20}]}.
```

Reply:
Hm if you want me to I can do it but I believe it amounts to the same.

Reply:
I guess, I'd prefer to be explicit about it.